.buildkite/pipeline.yml

```diff
@@ -254,19 +310,18 @@ steps:
         account_ids: "436617320021"
         no-include-email: true
         region: "us-west-2"
-      docker#v2.1.0:
+      docker#v3.2.0:
+        shell: ["/bin/bash", "-i", "-e", "-c"]
         debug: true
-        image: "436617320021.dkr.ecr.us-west-2.amazonaws.com/ci:ubuntu18_2-3"
+        image: "436617320021.dkr.ecr.us-west-2.amazonaws.com/ci:ubuntu18_2-4"
         propagate-environment: true
         workdir: /data/job
     timeout: 60
 
-  - command: |
-        echo "--- :arrow_down: Downloading Build Directory"
```

Comment:
Please don't delete these types of print statements.
Without this print statement, it will look like the agent has hanged.